src/bipartite.jl

```diff
@@ -0,0 +1,96 @@
+"""
+    BipartiteView{T<:Real} <: AbstractGraph{Int64}
+
+
+Creates a bipartite view of a hypergraph.
+Note this is a view - changes to the original hypergraph will be automatically reflected in the view.
+
+**Constructors**
```

Comment:
This format is discouraged (it later floods REPL). See e.g. how `Array` is documented. OTOH Examples section is useful to have.